lib/collection.js

```diff
@@ -791,7 +793,9 @@ Collection.prototype.replaceOne = function(filter, doc, options, callback) {
  * @param {(number|string)} [options.w] The write concern.
  * @param {number} [options.wtimeout] The write concern timeout.
  * @param {boolean} [options.j=false] Specify a journal write concern.
- * @param {Array} [options.arrayFilters] optional list of array filters referenced in filtered positional operators
+ * @param {array|object} [options.arrayFilters] optional list of array filters referenced in filtered positional operators
+ * @param {boolean} [options.bypassDocumentValidation=false] Allow driver to bypass schema validation in MongoDB 3.2 or higher.
+ * @param {object} [options.collation] Specify collation (MongoDB 3.4 or higher) settings for update operation (see 3.4 documentation for available fields).
```

Comment:
`replace` operation, not `update` operation

Reply:
This is for `updateMany`.

Reply:
concatenated again, but this comment should be move up to the previous section where it is documented as an `update` command but is for `replaceOne`